packages/contracts-ops/contracts/scripts/DeployAccountant.s.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "forge-std/Script.sol";
+import {Config} from "../Config.sol";
+import {JsonWriter} from "../JsonWriter.sol";
+
+import {AllowListNFTRecoveryAccountant} from "@nomad-xyz/contracts-bridge/contracts/accountants/NFTAccountant.sol";
+import {UpgradeBeacon} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeacon.sol";
+import {UpgradeBeaconProxy} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeaconProxy.sol";
+
+abstract contract DeployAccountant is Script, Config {
+    using JsonWriter for JsonWriter.Buffer;
+    using JsonWriter for string;
+
+    JsonWriter.File outputFile;
+
+    AllowListNFTRecoveryAccountant implementation;
+    UpgradeBeacon beacon;
+    UpgradeBeaconProxy proxy;
+
+    // entrypoint
+    function deploy(string calldata _configFile, string memory _domain, string memory _outputFile, bool _overwrite) public {
+        // initialize
+        __Config_initialize(_configFile);
+        _outputFile = string(abi.encodePacked("./actions/", _outputFile));
+        outputFile.path = _outputFile;
+        outputFile.overwrite = _overwrite;
+        // deploy & configure accountant
+        vm.startBroadcast();
+        deployAccountant(_domain);
+        vm.stopBroadcast();
+        // write contract addresses to JSON
+        write();
+    }
+
+    // Deploys & configures the NFTAccountant with upgrade setup
+    function deployAccountant(string memory _domain) internal {
+        address fundsRecipient = vm.addr(567); // TODO
+        address accountantOwner = vm.addr(89); // TODO
+        // deploy implementation
+        implementation = new AllowListNFTRecoveryAccountant(
+                address(bridgeRouter(_domain)),
+                fundsRecipient
+            );
+        // initialize implementation
+        implementation.initialize();
+        // deploy UpgradeBeacon (with UpgradeBeaconController as owner)
+        beacon = new UpgradeBeacon(
+            address(implementation),
+            address(upgradeBeaconController(_domain))
+        );
+        // deploy UpgradeBeaconProxy
+        proxy = new UpgradeBeaconProxy(
+            address(beacon),
+            "0x"
```

Comment:
I think you mean `""`

Reply:
https://github.com/nomad-xyz/monorepo/pull/501/commits/1a99637a989ff0a81c1ffc913aff1be2a5ff29af